frontends/ol-widgets/src/components/editing/ManageWidgetDialog.test.tsx

```diff
@@ -72,11 +72,12 @@ describe("ManageWidgetDialog (Editing)", () => {
   })
 
   it("passes classes to relevant elements", () => {
+    const fakerClassName = () => faker.unique(faker.lorem.slug)
```

Comment:
I added this because a test as being flaky, I *believe* because of collisions between the randomly generated classnames.